apps/examples/testcase/ta_tc/audio/itc/itc_audio_main.c

```diff
@@ -74,241 +63,754 @@ static void clean_all_data(int fd, char *buffer)
 }
 
 /**
-* @testcase         audio_pcm_open_close_tc_p
+* @testcase         itc_audio_pcm_open_close_p
 * @brief            open and initlaize pcm structure
 * @scenario         Open pcm and check its validation
 * @apicovered       pcm_open, pcm_close
 * @precondition     NA
 * @postcondition    NA
 */
-static void itc_audio_pcm_open_close_tc_p(void)
+static void itc_audio_pcm_open_close_p(void)
 {
+	g_pcm = NULL;
 	g_pcm = pcm_open(0, 0, PCM_IN, NULL);
+	TC_ASSERT_NEQ("pcm_open", g_pcm, NULL);
 	TC_ASSERT_GT("pcm_open", pcm_get_file_descriptor(g_pcm), 0);
```

Comment:
here the cleanup macro should be used and g_pcm should be closed in the cleanup function otherwise the next tc will fail due to error in this tc.

Reply:
@shivgarg 
There is no need to close g_pcm, as it is not yet open. This Assert checks whether pcm_open API is actually open or not, if it is not opened, then it cannot be closed

Reply:
pcm_close is still needed, since we need to free the pcm struct that is allocated. There is discrepancy in pcm_close regarding this but it would be settled soon.

Reply:
okay I will close g_pcm in cleanup function

Reply:
Is below check fine :
g_pcm = pcm_open(0, 0, PCM_IN, NULL);
TC_ASSERT_GT_CLEANUP("pcm_open", pcm_get_file_descriptor(g_pcm), 0, pcm_close(g_pcm));

Reply:
Yes, it is sufficient.